portfolio/src/main/webapp/script.js

```diff
@@ -204,23 +204,22 @@ function loadFavoritePlaces() {
 }
 
 function makeChart() {
-  const dataset = new google.visualization.DataTable();
-  dataset.addColumn('string', 'Books');
-  dataset.addColumn('number', 'Re-reads');
-  dataset.addRows([
-    ['The Book Thief', 3],
-    ['Harry Potter', 7],
-    ['The Mark of Athena', 8]
-  ]);
-
-  const parameters = {
-    'title': 'Favorite Books',
-    'width': 500,
-    'heigth': 400,
-    'backgroundColor': '#E6E6FA',
-    'pieHole': 0.3,
-  };
-
-  const pieChart = new google.visualization.PieChart(document.getElementById('chart'));
-  pieChart.draw(dataset, parameters);
+  fetch('/chart').then(response => response.json()).then((covidCases) => {
+    var listData = []
+    listData.push(['State', 'Number of Cases', 'Number of Deaths']);
+    covidCases.forEach((entry) => {
+      listData.push([entry.state, entry.cases, entry.deaths]);
+    });
+    const dataset = new google.visualization.arrayToDataTable(listData);
+    
+    const parameters = {
+      chart: {
+      'width':900,
+      'height':500},
```

Comment:
nit: space after :
nit: put the } on its own line if the first { is followed by a hard return

Reply:
Fixed the nits, thank you for the comments!